virttest/qemu_devices/qcontainer.py

```diff
@@ -304,7 +304,7 @@ def __eq__(self, qdev2):
         qdev2 = qdev2.__dict__
         for key, value in six.iteritems(self.__dict__):
             if key in ("_DevContainer__devices", "_DevContainer__buses",
-                       "_DevContainer__state",
+                       "_DevContainer__state", "_DevContainer__caps",
```

Comment:
After https://github.com/avocado-framework/avocado-vt/pull/2274/commits/3f0da17ed7830c00fa20575defdb43aed1d90c2d got merged, it should be `"caps"` I guess.

Reply:
Hi @luckyh, I am not aware of these most recent changes. Do you mean renaming the key `"_DevContainer__caps"` simply to `"caps"`? Unfortunately, we only test official release versions so I won't be able to test this any time soon. We could still proceed with this if you can confirm that this is indeed the correct key now.

Reply:
As capabilities used to be a private attribute, you can use the name `"_DevContainer__caps"` to access it, but it is a public one now, therefore have to change the name to `"caps"`.

Reply:
Alright, I have pushed an adapted version of this.